File: Sorting/selectionSort.py
def selectionSort(arr):
    for i in range(len(arr)):
        min_index = i
        for j in range(i+1,len(arr)):
            if arr[j] < arr[min_index]:
                min_index = j
        arr[i], arr[min_index] = arr[min_index], arr[i]

def printList(arr):
    for i in range(0,len(arr)):
        print(arr[i],end=" ")

if __name__ == '__main__':
    li = [6,3,5,8,9,0]
    selectionSort(li)
    printList(li)

File: Algorithms/src/com/company/SelectionSort.java
package com.company;

import java.util.Scanner;

public class SelectionSort {
    public static void main(String[] args) {
        int n;
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        int arr[] = new int[n];

        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }
        int minIndex = 0;
        for (int i = 0; i < n; i++) {
            minIndex = i;
            for (int j = i; j < n; j++) {
                if (arr[minIndex] > arr[j]) {
                    minIndex = j;
                }

            }
            int temp = arr[minIndex];
            arr[minIndex] = arr[i];
            arr[i] = temp;
        }
        for(int e:arr){
            System.out.print(e+" ");
        }

    }
}
